images/runtime/php/7.4/base.bullseye.Dockerfile

```diff
@@ -19,6 +19,12 @@ RUN echo -e '<FilesMatch "\.(?i:ph([[p]?[0-9]*|tm[l]?))$">\n SetHandler applicat
 # Disable Apache2 server signature
 RUN echo -e 'ServerSignature Off' >> /etc/apache2/apache2.conf
 RUN echo -e 'ServerTokens Prod' >> /etc/apache2/apache2.conf
+RUN { \
+   echo '<DirectoryMatch "^/.*/\.git/">'; \
+   echo '   Order deny,allow'; \
+   echo '   Deny from all'; \
+   echo '</DirectoryMatch>'; \
+} >> /etc/apache2/apache2.conf
```

Comment:
Does this command required when migrating to bullseye, or it is a missing part generally?

Reply:
As discussed offline, this change resulted from running the `buildRuntimeBaseImages.sh` script.